src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,31 @@
+package duke.command;
+
+import duke.task.Deadline;
+import duke.main.DukeException;
+import duke.main.TaskList;
+import java.time.DateTimeException;
+import java.time.LocalDate;
+
+public class DeadlineCommand extends AddCommand {
+    private String description;
+    private String date;
+
+    public DeadlineCommand(String description, String date) {
+        this.description = description;
+        this.date = date;
+    }
+
+    /**
+     * Add new tasks.
+     *
+     * @param taskList the list of tasks
+     */
```

Comment:
I noticed this javadoc repeated throughout the AddCommand child classes. Putting it in the parent class and using @inheritDoc is a good alternative!